index.d.ts

```diff
@@ -31,7 +31,7 @@ interface ListOptions {
 }
 
 export interface FastifyStaticOptions {
-  root: string;
+  root: string | string[];
```

Comment:
we also test our types, please add a test for this type as well

Reply:
Added a small addition to the `test/types/index.ts` file to check the type works.